src/ROUTES.js

```diff
@@ -85,9 +85,9 @@ export default {
     IOU_SEND_ADD_BANK_ACCOUNT: `${IOU_SEND}/add-bank-account`,
     IOU_SEND_ADD_DEBIT_CARD: `${IOU_SEND}/add-debit-card`,
     IOU_SEND_ENABLE_PAYMENTS: `${IOU_SEND}/enable-payments`,
-    getIouRequestCurrencyRoute: (reportID) => `${IOU_REQUEST_CURRENCY}/${reportID}`,
-    getIouBillCurrencyRoute: (reportID) => `${IOU_BILL_CURRENCY}/${reportID}`,
-    getIouSendCurrencyRoute: (reportID) => `${IOU_SEND_CURRENCY}/${reportID}`,
+    getIouRequestCurrencyRoute: (reportID, currCurrency, backTo) => `${IOU_REQUEST_CURRENCY}/${reportID}?currency=${currCurrency}&backTo=${backTo}`,
+    getIouBillCurrencyRoute: (reportID, currCurrency, backTo) => `${IOU_BILL_CURRENCY}/${reportID}?currency=${currCurrency}&backTo=${backTo}`,
+    getIouSendCurrencyRoute: (reportID, currCurrency, backTo) => `${IOU_SEND_CURRENCY}/${reportID}?currency=${currCurrency}&backTo=${backTo}`,
```

Comment:
why it's called as `currCurrency`?

it can be `currency` or `currencyCode` right?

Reply:
Stands for current currency. 😄

I will change it to `currency`.

Reply:
Done